### ISSUE TITLE
QW_CLUSTER_ENDPOINT cannot be customized

### ISSUE THREAD

**Issue by splix** [quickwit-oss/helm-charts#108]
In `_helpers.tpl` there is the following lines:
```
{{- define "quickwit.environment" -}}
....
- name: QW_CLUSTER_ENDPOINT
  value: http://{{ include "quickwit.fullname" $ }}-metastore.{{ $.Release.Namespace }}.svc.cluster.local:7280
{{- range $key, $value := .Values.environment }}
- name: "{{ $key }}"
  value: "{{ $value }}"
{{- end }}
{{- end }}
```

Which hardcodes the cluster name as `cluster.local` which is not always the case, and so Quickwit doesn't work in clusters with non-default name. 

It's impossible to override the value by specifying an `.environment` because Kubernetes takes the first value in the `env` list. So, right now it allows only to _append_ a variable.

I see at least 3 ways how it can be fixed:
1. Allows to set the cluster name. That's probably most common in other charts
2. Do not use the cluster name at all `http://{{ include "quickwit.fullname" $ }}-metastore.{{ $.Release.Namespace }}:7280` would suffice since it's not supposed to connect to an endpoint in another cluster.
3. Move the last part (i..e, the `range` loop) to the top, which would allow to update any environment variable, not just append.

**Comment by splix** [comment]
Yes, I can do that

**Comment by guilload** [comment]
Closed via #110.
